src/components/HoverHandler/index.js

```diff
@@ -2,34 +2,65 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 
 
+const DELAY = 300
```

Comment:
What do you think of having a prop (with the default value 300) for the delay?

Reply:
thought about that, didn't really see a need